backend/services/reaction.py

```
@@ -0,0 +1,77 @@
+
+
+def add_like(like_data: LikeCreate, db: Session, current_user: User):
+    created_at = datetime.now(ZoneInfo("UTC"))
+    new_like = Like(user_id=current_user.id, post_id=like_data.post_id, comment_id=like_data.comment_id, created_at=created_at)
+    db.add(new_like)
+    update_like_count(like_data, db, "add")
+    db.commit()
+    db.refresh(new_like)
+    return new_like
+
+def get_like_count(db: Session, like_data: LikeCreate):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_like_count`

(ANN201)